### PR TITLE
Added some icons

### PR DESCRIPTION
**Pull request by joelchrono12**
showcase

### CODE REVIEW

**Comment by joelchrono12** [comment]
I fixed #774, not #744, my bad

**Comment by joelchrono12** [comment]
showcase

**Comment by joelchrono12** [comment]
showcase

File: other/joelchrono12/marvelapp.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<svg width="48" height="48" version="1.1" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
+ <defs>
+  <style>.a{fill:none;stroke:#fff;stroke-linecap:round;stroke-linejoin:round;}</style>
```
Comment (aloxe) [inline other/joelchrono12/marvelapp.svg:3]: why this is duplicated?

**Comment by Donnnno** [comment]
The icons look great as always! Just gonna do some cleanup here and there, like this one:

image

Kinda dirty anchor points

**Pull request merged**